src/utils/constants.rs

```diff
@@ -90,6 +90,9 @@ fn agency_map() -> HashMap<&'static str, &'static str> {
     res.insert("eurockot", "Eurockot Launch Services");
     res.insert("rocketlab", "Rocket Lab Ltd");
     res.insert("orbitalatk", "Orbital ATK");
+    res.insert("casc", "China Aerospace Science and Technology Corporation");
```

Comment:
```suggestion
    res.insert("casc", "China Aerospace Corporation");
```

Reply:
seems that LL uses the shorter name in their api, so that's why I requested this change